src/Type/Doctrine/Descriptors/SmallFloatType.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Doctrine\Descriptors;
+
+class SmallFloatType extends FloatType
```

Comment:
No inheritance please

Reply:
Are you sure? I believe SmallFloatType IS FloatType in terms of type descriptor.